source/game_sa/Tasks/TaskTypes/CTaskSimpleSetStayInSamePlace.cpp

```diff
@@ -0,0 +1,32 @@
+#include "StdInc.h"
+
+void CTaskSimpleSetStayInSamePlace::InjectHooks()
+{
+    ReversibleHooks::Install("CTaskSimpleSetStayInSamePlace", "Constructor", 0x62F590, &CTaskSimpleSetStayInSamePlace::Constructor);
+    //VTABLE
+    ReversibleHooks::Install("CTaskSimpleSetStayInSamePlace", "ProcessPed", 0x62F5E0, &CTaskSimpleSetStayInSamePlace::ProcessPed_Reversed);
+}
+
+CTaskSimpleSetStayInSamePlace* CTaskSimpleSetStayInSamePlace::Constructor(bool bStayInSamePlace)
+{
+    this->CTaskSimpleSetStayInSamePlace::CTaskSimpleSetStayInSamePlace(bStayInSamePlace);
+    return this;
+}
+
+// 0x62F590
+CTaskSimpleSetStayInSamePlace::CTaskSimpleSetStayInSamePlace(bool bStayInSamePlace)
+{
+    m_bStayInSamePlace = bStayInSamePlace;
+}
+
+// 0x62F5E0
+bool CTaskSimpleSetStayInSamePlace::ProcessPed(CPed* ped)
+{
+    return ProcessPed_Reversed(ped);
+}
+
+bool CTaskSimpleSetStayInSamePlace::ProcessPed_Reversed(CPed* ped)
+{
+    ped->bStayInSamePlace = m_bStayInSamePlace;
+    return true;
+}
```

Comment:
Placing ```*_Reversed``` methods at the bottom will reduce refactoring time in the future.

```suggestion
bool CTaskSimpleSetStayInSamePlace::ProcessPed(CPed* ped)
{
    ped->bStayInSamePlace = m_bStayInSamePlace;
    return true;
}

bool CTaskSimpleSetStayInSamePlace::ProcessPed_Reversed(CPed* ped)
{
    CTaskSimpleSetStayInSamePlace::ProcessPed(ped);
}
```

Reply:
I think it's fine. Not a big deal since I'll automate the process by writing a tool for it.